interactives/high-score-boxes/css/style.css

```diff
@@ -0,0 +1,59 @@
+
+body {
+}
+
+/* covering boxes */
+div.box {
+    width: 100px;
+    height: 100px;
+    /*border: 10px;*/
```

Comment:
It would be nice to delete unused lines from files (unless it may be used in the future).